robot_skills/test/test_move_down_until_force_sensor_edge_up

```diff
@@ -0,0 +1,9 @@
+#!/usr/bin/env python
```

Comment:
- Filename extension would be nice
- hardware tests should be seperated from CI tests,